06-building-a-program-start-to-finish/automating-tasks/output/README.md

```diff
@@ -0,0 +1,14 @@
+# Automatin Tasks
+
+## Output
+- **Using**: Gulp
+- **Deliver**: a screencast showing gulp watching and refreshing your browser when changes are made to your HTML, CSS & JS files
+- **With**: *independently*
+
+## Link to GitHub Repo
+
+[My InvertedIndex Repo](https://github.com/andela-opikuda/Invertedindex/ "Visit Damisi's Project Repo")
+
+## Link to Screencast
+
+[My Screencast](https://www.youtube.com/watch?v=fiv9PO3LmD0 "Visit Damisi's Screencast")
```

Comment:
I'd have loved to see you watch for changes in a separate JS file related to your Fronnt-end and not your gulp file but you get the idea. I'll let it pass...